learn/react-native/mobile-build-manual.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "React Native - Mobile Manual Build"
+id: "mobile-build-manual"
+---
+---
+
+In application development process, you create iOS Package App Store (IPA) or Android Package Kit (APK) files for developing any mobile application. IPA file is the compressed version of the required files and resources of IOS application, created to test the IOS application before it is live. APK file is the similar package file used in Android devices.
```

Comment:
After the mobile app is developed, creating ios... and android... is necessary for deploying it on the app store.